routes/web.php

```diff
@@ -27,29 +27,9 @@
 /**
```

Comment:
this is still in web file, when you are moving queries to Controller
`Route::get('/', function () {
    $tasks = Task::orderBy('created_at', 'asc')->get();
    return view('tasks', [
        'tasks' => $tasks
    ]);
});`